exporters/metric/prometheus/prometheus_test.go

```diff
@@ -69,6 +74,8 @@ func TestPrometheusExporter(t *testing.T) {
 	expected = append(expected, `valuerecorder_sum{A="B",C="D",R="V"} 19.6`)
 
 	compareExport(t, exporter, expected)
+	time.Sleep(3 * time.Second)
```

Comment:
this will make the tests take 3 seconds longer, is there another mechanism of forcing this e.g. by advancing a fake clock or do we really need the sleep?

Reply:
Thanks. I need to understand the caching mechanism used by the Prometheus server to debounce requests or supply a testable clock.

Reply:
Looks like I was mistaken. The test would fail without the sleep in the old code and I've removed it.